src/main/_docs/use-che-as-an-ide/ide-ssh.md

```diff
@@ -20,7 +20,10 @@ We then provide various client utilities (authored in Docker!) for connecting to
 # List Workspaces  
 You can get a list of workspaces in a Che server that have an SSH agent deployed. These are the workspaces that you can SSH into.
 ```shell  
-$ che action list-workspaces
+$ docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock
```

Comment:
-1:
- users are sometime not reading the whole page.
- users will not understand, why sometime it has full syntax and sometime not